VERSION_NUMBER

```diff
@@ -1 +1 @@
-1.9.0
+1.9.100
```

Comment:
I suggest changing this to 1.10.0 in master.... In release branch create a new PR to just change the version in this file to 1.9.100

The idea is any new PR going into master post this PR is merged will be towards the next release.

Reply:
Thanks - will use 1.9.100 only on the release branch.